modules/ROOT/pages/administration/databases.adoc

```diff
@@ -903,6 +925,76 @@ ALTER DATABASE nonExisting IF EXISTS SET TOPOLOGY 1 PRIMARY 0 SECONDARY
 [role="statsonlyqueryresult"]
 0 rows
 
+[role=enterprise-edition]
+[[administration-databases-alter-database-options]]
+=== Options
+
+The `ALTER DATABASE` command can be used to set or remove specific options.
+
+[NOTE]
+====
+There are no available `OPTIONS` values for composite databases.
+====
+
+
+[options="header"]
+|===
+
+| Key | Value | Description
+
+| `txLogEnrichment`
+| `FULL` \| `DIFF` \| `NONE`
+|
+Defines level of enrichment applied to transaction logs for Change Data Capture (CDC) purposes.
+
+|===
+
+.+ALTER DATABASE SET OPTION+
+======
+
+.Query
+[source, cypher, test-skip]
+----
+ALTER DATABASE `options-example`
+SET OPTION txLogEnrichment 'FULL'
+----
+
+======
+
+.+ALTER DATABASE REMOVE OPTION+
+======
+
+.Query
+[source, cypher, test-skip]
+----
+ALTER DATABASE `options-example`
+REMOVE OPTION txLogEnrichment
+----
+
+======
+
+
+.+SHOW DATABASE+
+======
+
+.Query
+[source, cypher, test-skip]
+----
+SHOW DATABASES yield name, options
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| +name+ | +options+
+| +"options-example"+ | +{txLogEnrichment: "FULL"}+
```

Comment:
Either this show example should be before the remove or it feels weird to have anything in the options map. Might actually be nice to do the SET, then SHOW, then REMOVE and then SHOW again to show that it gets removed (or if it falls back to default values then the SET should set something that isn't the default value so we can see the difference)